CryptoPrices.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
+  "object": {
+    "pins": [
+      {
+        "package": "Pilot",
+        "repositoryURL": "https://github.com/Thieurom/Pilot",
+        "state": {
+          "branch": null,
+          "revision": "d0c990223f3447fd912ae716d03676fd1db714af",
+          "version": "0.5.1"
+        }
+      }
+    ]
+  },
+  "version": 1
+}
```

Comment:
Let's try out the `Pilot` library for this project 👍 I would also like to experiment more on `Moya` but yeah maybe another project 🙈

Reply:
@minhnimble 🙇 
`Pilot` borrows the very idea of how to make the network layer by describing the API specs (blueprint), and the usage is quite similar. `Pilot` supports `async/await` while `Moya` does not (yet) (It supports `RxSwift` and `Combine` though). It also is very simple, lightweight, no dependency (`Moya` depends on `Alamofire`).
So let's the `Pilot` fly us 🚀